classes/collectors/ReactionCollectorBase.js

```diff
@@ -295,12 +295,12 @@ export class ReactionCollectorBase {
      * @param {User} args.user Discord user
      * @param {string} args.pog "Proof of good" - message to display in GP history
 	 */
-	async awardApprovalGP({user, pog}) {
+	async awardApprovalGP({user, approver, pog}) {
 		const { bot, msg } = this;
 
 		await Firebase.awardPoints(await Firebase.getLeylineUID(user.id), this.APPROVAL_GP, {
 			category: pog,
-			comment: `User's Discord ${this.media_type} (${msg.id}) was approved by ${user.tag}`,
+			comment: `User's Discord ${this.media_type} (${msg.id}) was approved by ${approver}`,
```

Comment:
```suggestion
			comment: `User's Discord ${this.media_type} (${msg.id}) was approved by ${approver.tag}`,
```
see above comment; pass in the entire object, then reference a property